### PR TITLE
checking if forker's dependencies were installed before start

### PR DESCRIPTION
**Pull request by netojoaobatista**
Just a little check for composer.lock before start forker.

### CODE REVIEW

File: bin/forker.php [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+$forkerDirectory = getcwd();
+
+if (basename($forkerDirectory) === 'bin') {
+    $forkerDirectory = dirname($forkerDirectory);
+}
+
+if (!is_file('composer.lock')) {
+    echo 'You must run composer install before use forker' . PHP_EOL;
+    exit;
+}
```
Comment (henriquemoody) [inline bin/forker.php:8]: IMO will be better you use:

``` php
$forkerDirectory = __DIR__ . '/../';
```

I already created a symlink for `forker` on my $PATH, the current work directory is not gonna work in all scenarios.

Comment (netojoaobatista) [inline bin/forker.php:8]: Yes, you're right!

Better yet, dirname(**DIR**) will bring us to forker's installation directory.

Comment (henriquemoody) [inline bin/forker.php:8]: Sure, something like:

``` php
$composerLockFile = dirname(__DIR__) . '/composer.lock';
```

**Comment by netojoaobatista** [comment]
It's done. Also fixed the same problem with bash script.

**Comment by henriquemoody** [comment]
:sunglasses:

**Pull request merged**